### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by cyanbeast01**
added marble mania autosplitter
my coding is terrible hope nothing go wrong

If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -21913,13 +21913,13 @@
     </AutoSplitter>
     <AutoSplitter>
         <Games>
-            <Game>Awesome Tanks</Game>
+            <Game>Marble Mania</Game>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:21916]: Why did you remove this other autosplitter?

File: LiveSplit.AutoSplitters.xml [hunk 2]
```diff
@@ -21913,13 +21913,13 @@
...
         </Games>
         <URLs>   
-            <URL>https://raw.githubusercontent.com/Biksell/asl/main/Awesome%20Tanks/AwesomeTanks.asl</URL>
+            <URL>https://github.com/cyanbeast01/Marble-Mania/blob/main/Marble-Mania-livesplliter(buggy).asl</URL>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:21919]: This needs to be a raw URL. Also, if it's buggy, then don't add it here.

File: LiveSplit.AutoSplitters.xml (hunk 1 above)
Comment (cyanbeast01) [reply LiveSplit.AutoSplitters.xml:21916]: because i copy and pas te

File: LiveSplit.AutoSplitters.xml (hunk 2 above)
Comment (cyanbeast01) [reply LiveSplit.AutoSplitters.xml:21919]: i guess
